packages/core/src/App/Containers/Layout/app-contents.jsx

```diff
@@ -68,21 +111,24 @@ AppContents.propTypes = {
     is_positions_drawer_on: PropTypes.bool,
     is_route_modal_on: PropTypes.bool,
     pwa_prompt_event: PropTypes.object,
-    setPWAPromptEvent: PropTypes.func,
 };
 
 export default withRouter(
-    connect(({ ui, segment }) => ({
-        // is_logged_in          : client.is_logged_in,
-        // addNotificationBar    : ui.addNotificationBar,
+    connect(({ client, gtm, segment, ui }) => ({
+        is_eu_country: client.is_eu_country,
+        is_logged_in: client.is_logged_in,
+        is_tracking: gtm.is_tracking,
```

Comment:
Since this no longer exists, we can remove it, right?

```suggestion
```

Reply:
Ah sure.